wasm/prover/tests/web.rs

```diff
@@ -0,0 +1,40 @@
+//! Test suite for the Web and headless browsers.
+
+#![cfg(target_arch = "wasm32")]
+
+extern crate wasm_bindgen_test;
+use std::str;
+use wasm_bindgen_test::*;
+
+extern crate tlsn_extension_rs;
+use tlsn_extension_rs::*;
+
+macro_rules! log {
+    ( $( $t:tt )* ) => {
+        web_sys::console::log_1(&format!( $( $t )* ).into());
+    }
+}
+
+wasm_bindgen_test_configure!(run_in_browser);
+
+#[wasm_bindgen_test]
+fn dummy_test() {
+    assert_eq!(1 + 1, 2);
+}
+
```

Comment:
Is this `dummy_test` needed?

Reply:
No, this was an intermediate step to make sure `wasm_bindgen_test` worked 😊